cmd/deploy/deploy.go

```diff
@@ -209,6 +227,8 @@ func (dc *deployCommand) runDeploy(ctx context.Context, cwd string, opts *Option
 		fmt.Sprintf("%s=true", model.OktetoDisableSpinnerEnvVar),
 		// Set BUILDKIT_PROGRESS=plain env variable, so all the commands disable docker tty builds
 		fmt.Sprintf("%s=true", model.BuildkitProgressEnvVar),
+		// Set OKTETO_NAMESPACE=namespace-name env variable, so all the commandsruns on the same namespace
+		fmt.Sprintf("%s=%s", model.OktetoNamespaceEnvVar, opts.Namespace),
```

Comment:
Shouldn't this be `okteto.Context().Name` to use the same namespace we are using in the kubeconfig?

Reply:
yes, you're right

Reply:
yep, I didn't copy the whole sentence xDD